hub/src/main/java/org/motechproject/hub/web/IntentVerificationThreadRunnable.java

```diff
@@ -223,6 +224,9 @@ public void run() {
         } catch (RuntimeException e) {
             LOGGER.error("An error occurred during intent verification", e);
             statusLookup = SubscriptionStatusLookup.INTENT_FAILED;
+        } catch (HttpException e ) {
```

Comment:
Since this does exactly the same as above, merge this with the other catch (multi-catch)